### PR TITLE
[Graduation] Modify the graduation version

### PR DESCRIPTION
**Pull request by CalvinKirs**
Remove DISCLAIMER
rename release package name

### CODE REVIEW

**Review by chengshiwen — CHANGES_REQUESTED** [review]
@CalvinKirs If you only search `incubat`, there will be a lot of relevant results

**Comment by CalvinKirs** [comment]
> @CalvinKirs If you only search `incubat`, there will be a lot of relevant results



> @CalvinKirs If you only search `incubat`, there will be a lot of relevant results

I checked, not all of them need to be modified.

File: LICENSE [hunk 1]
```diff
@@ -200,9 +200,9 @@
    See the License for the specific language governing permissions and
    limitations under the License.
 =======================================================================
-Apache DolphinScheduler (incubating) Subcomponents:
+Apache DolphinScheduler  Subcomponents:
```
Comment (chengshiwen) [inline LICENSE]: I have checked and there is no problem.
It would be better if the extra space could be removed.

**Review by chengshiwen — CHANGES_REQUESTED** [review]

**Review by chengshiwen — APPROVED** [review]
+1

**Review by davidzollo — APPROVED** [review]
+1
good job

**Pull request merged**